### PR TITLE
feat: improve typing for color packages

### PR DESCRIPTION
**Pull request by tlebon**
## Pull Request Checklist

- [ ] My code is covered by tests
- [ ] I will [merge the PR as breaking change](https://github.com/wireapp/wire-web-packages/wiki/Releases#create-major-release), if the API contract changes

Something that has bothered me for ages: our COLOR variables are woefully untyped. 

Before: 

https://user-images.githubusercontent.com/37285713/222223047-5dacec87-fc0e-4612-a777-3fc7ef135f51.mov

After: 

https://user-images.githubusercontent.com/37285713/222223093-e90361cb-43b4-44ad-a6a7-91247c56f66b.mov

### CODE REVIEW

File: packages/react-ui-kit/src/Identity/colors-v2.ts [hunk 1]
```diff
@@ -26,7 +26,7 @@ export const BASE_LIGHT_COLOR: Record<string, string> = {
   TURQUOISE: '#01718e',
 };
```
Comment (atomrc) [inline packages/react-ui-kit/src/Identity/colors-v2.ts:27]: Super nice one. Let's be even more strict and add `as const` at the end ;)

Comment (aweiss-dev) [inline packages/react-ui-kit/src/Identity/colors-v2.ts:27]: For Stuff like that (Design Tokens, Colors, etc), I recommend exporting as const.

This improves the code completion a lot :).

https://dev.to/aexol/typescript-tutorial-use-as-const-to-export-colors-39fl

**Review by aweiss-dev — CHANGES_REQUESTED** [review]

**Review by atomrc — APPROVED** [review]

**Review by PatrykBuniX — APPROVED** [review]
😍

File: packages/react-ui-kit/src/Identity/colors-v2.ts (hunk 1 above)
Comment (tlebon) [reply packages/react-ui-kit/src/Identity/colors-v2.ts]: Plz unblock

**Pull request merged**